### PR TITLE
Update Ruby version

### PR DESCRIPTION
**Pull request by mchowning**
Allows configureApply task to work on Apple Silicon

Making this change at the suggestion of @jkmassel  and @AliSoftware . I'm not very familiar with ruby and how we're using it, so I honestly don't have a sense for the impact of this change. In other words, you shouldn't trust that I know what I'm doing here. 🙂 

Before this change, running the `configureApply` task on my M1 macbook gave me this error:

```
ld: warning: ignoring file /opt/homebrew/Cellar/xz/5.2.5/lib/liblzma.dylib,
building for macOS-x86_64 but attempting to link with file built for macOS-arm64
Undefined symbols for architecture x86_64:
  "_lzma_auto_decoder", referenced from:
      _xz_make in libxml2.a(xzlib.o)
  "_lzma_code", referenced from:
      _xz_decomp in libxml2.a(xzlib.o)
  "_lzma_end", referenced from:
      ___libxml2_xzclose in libxml2.a(xzlib.o)
  "_lzma_properties_decode", referenced from:
      _xz_make in libxml2.a(xzlib.o)
ld: symbol(s) not found for architecture x86_64
```

To test:

Run the configureApply task: `./gradlew configureApply`

## Regression Notes
1. Potential unintended areas of impact
This is a very good question. I don't know.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
I ran `./gradlew configureApply` successfully on an Apple Silicon Macbook.

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:

- [X] I have completed the Regression Notes.
- [X] I have considered adding accessibility improvements for my changes.
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.6.4
+2.7.2
```
Comment (jkmassel) [inline .ruby-version]:
```suggestion
2.7.4
```

Let's bump this to `2.7.4` (as discussed in Slack), then :shipit: 

Thanks for taking care of this!!

**Review by jkmassel — APPROVED** [review]

**Review by AliSoftware — APPROVED** [review]
Overall that looks good 👍  and CI still passes with the new ruby, so we're good!
Thanks for tackling that 🙇 

PS: I wasn't sure if this might need us to regenerate e.g. the `Gemfile.lock` to update some checksums but didn't get any changed file in git after running `bundle install`, so seems not!

Note: I don't have an M1 to test `configureApply` on it with these changes so only tested it on my Intel Mac.

**Review by ealeksandrov — APPROVED** [review]
I've followed README for fresh Android setup and then `./gradlew configureApply` succeeded for me on M1 ✅

**Comment by mchowning** [comment]
For some reason, I had to clean out all of my gems and reinstall them to get `configureApply` to work when I switched to 2.7.4. Not sure how I got that messed up, but everything seems to be working fine now.

**Pull request merged**